Dockerfile.node_sidecar

```diff
@@ -1,4 +1,4 @@
-FROM python:3.11 AS builder
+FROM python:2 AS builder
```

Comment:
Why are you moving sidecar to python 2?

Reply:
A typo :see_no_evil: Thanks!

To be clear, this PR is still very much a WIP. Still nice to get the comment, I might have missed this (until it exploded while building the image after merging to main)